scintellometry/meta/observations.py

```diff
@@ -61,17 +60,27 @@ def gmrt_rawfiles(file_fmt, fnbase, **kwargs):
     return a 2-tuple for GMRT observation 'key':
     (timestamp file, [file1, ...])
 
-    Number of files depends on whether there is a ``nodes`` keyword argument
+    Number of files depends on whether there is a ``feeds`` keyword argument
     """
-    nodes = kwargs.get('nodes', None)
-    if nodes is None:
+    from scintellometry.phasing.GMRTNaming import getNode,getVolt
```

Comment:
I think it is somewhat more logical to have this as part of `io.gmrt` -- it is part of understanding where files wind up.